src/lib/operations/create-user/create-user.handler.ts

```diff
@@ -0,0 +1,42 @@
+import express from "express";
+import { createUser } from "./createUser";
+import { ExpressRequestWithToken } from "../../utils/auth/auth0";
+import { logRequestSuccess, logRequestStart } from "../../utils/log/log.utils";
+import { Route } from "../../server/server.constants";
+import { createOrPropagateError } from "../../server/errors/errors.utils";
+import { OthentErrorID } from "../../server/errors/error";
+
+export interface CreateUserIdTokenData {
+  fn: "createUser";
+  importOnly?: boolean;
+}
+
+export interface CreateUserResponseData {
+  data: boolean;
+};
+
+export function createUserHandlerFactory() {
+  return async (req: ExpressRequestWithToken<CreateUserIdTokenData>, res: express.Response) => {
+    const { idToken } = req;
+
+    // TODO: Only in the new version (old one didn't have data for createUser):
+    // const { data } = idToken; // || !data || data.fn !== "createUser"
```

Comment:
TODO: Update validation:

- New SDK: Must have a matching `data.fn` and no `keyName`.
- Old SDK: Must have `keyName` and no `data.fn`, but we'll ignore the value.